src/hooks/useActionsCount.ts

```diff
@@ -0,0 +1,61 @@
+import { startOfDay, subDays } from 'date-fns';
+
+import { useGetTotalColonyActionsQuery } from '~gql';
+import { getDomainDatabaseId } from '~utils/databaseId';
+
+import { getBaseSearchActionsFilterVariable } from './useActivityFeed/helpers';
+import useColonyContext from './useColonyContext';
+
+const getThirtyDaysAgoIso = () => {
+  const thirtyDaysAgo = subDays(new Date(), 30);
+  const midnightThirtyDaysAgo = startOfDay(thirtyDaysAgo);
+  return midnightThirtyDaysAgo.toISOString();
+};
+
+interface UseActionsCountParams {
+  domainId?: number;
+  onlyRecent?: boolean;
+}
+
+/**
+ * Hook returning the total number of actions matching the given params
+ */
+const useActionsCount = (params?: UseActionsCountParams) => {
+  const { domainId, onlyRecent } = params ?? {};
+  const { colony } = useColonyContext();
+  const { colonyAddress = '' } = colony ?? {};
+
+  const { data: totalActionData } = useGetTotalColonyActionsQuery({
+    variables: {
+      filter: {
+        ...getBaseSearchActionsFilterVariable(colonyAddress),
+        or: domainId
+          ? [
+              {
+                fromDomainId: {
+                  eq: getDomainDatabaseId(colonyAddress, domainId),
+                },
+              },
+              {
+                motionDomainId: { eq: domainId },
+              },
+            ]
+          : undefined,
```

Comment:
All this logic really depends on when we consider an action to be in a particular domain. Currently, this will count:
- All actions with `fromDomainId` matching the selected domain ID
- All motions created in the selected domain

In contrast, the logic determining the most active team only considers the `fromDomainId` field, and not motions.

I think it's one for @arrenv to confirm but I don't want to block this over a small detail.